test/threads.processor.test.ts

```
@@ -0,0 +1,96 @@
+    id: '63398805919',
+    username: 'spiramidgareorzea',
+    fullName: 'Gem',
+    pic: 'https://cdn.example/p.jpg'
+  };
+
+  it('maps text_fragments plaintext to status text', () => {
+    const body = 'Hello threads';
+    const post: Record<string, unknown> = {
+      pk: '3882494318431583186',
+      code: 'DXhZAMkljvS',
+      user: { pk: '63398805919', id: '63398805919', username: 'spiramidgareorzea', full_name: 'Gem' },
```

ESLint

[error] Replace `·pk:·'63398805919',·id:·'63398805919',·username:·'spiramidgareorzea',·full_name:·'Gem'` with `⏎········pk:·'63398805919',⏎········id:·'63398805919',⏎········username:·'spiramidgareorzea',⏎········full_name:·'Gem'⏎·····`

(prettier/prettier)